src/main/java/seedu/todo/commons/util/TimeUtil.java

```diff
@@ -0,0 +1,101 @@
+package seedu.todo.commons.util;
+
+import java.time.Clock;
+import java.time.Duration;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Utility methods that deals with time.
+ */
+public class TimeUtil {
+    
+    /* Constants */
+    private static final String DEADLINE_PREFIX_IN = "in ";
```

Comment:
Embedding the whitespace characters directly into the strings feels quite error prone. Perhaps a better solution would be to use a `StringJoiner` instead